config/config.yml

```diff
@@ -1,2 +1,6 @@
 broker: pyamqp://guest@localhost//
-backend: rpc://
\ No newline at end of file
+backend: rpc://
+montagu_url: http://localhost:8080
+montagu_user: test.user@example.com
+montagu_password: password
```

Comment:
Yeah, I think we said we'd store the task queue's Montagu credentials in the vault, and create the Montagu user during the deployment of the task queue component, as well as setting in this config file. Not sure what you mean by the relevant containers?

Reply:
If that configuration file is ready by something in a container, just how we'll deal with the bootstrapping process (start the container, inject secrets, start service, etc)

Reply:
Yeah so I guess we need to make sure this component doesn't try to run any tasks before it's ready ie before it's had its config copied in. I *think* that should just be a case of always bringing down the message queue container with the tasks container (which takes this config), and making sure that we don't bring up the mq until after the tasks container is all sorted.
I'm pretty sure if you bring up the tasks container before the mq is there it just complains a bit and waits for it to be available rather than erroring but I'll double-check. 
I'm not sure how it will deal with the containers being brought down while running a task. I suspect it will restart when they come back up, which is probably ok, but I'll check that too.